docs/upgrading.md

```diff
@@ -0,0 +1,111 @@
+Fedora and Ubuntu), upgrading is possible with some caveats:
+
+- Configuration files are not migrated (see [Configuration files](#configuration-files)).
+- It is necessary to perform a few cleanup steps as outlined in the
+  subsections below.
+
+Once the cleanup steps have been performed, simply follow the 3.x
+installation guides and any existing packaged versions of the 2.x
+components will be removed and the new 3.x components installed in their
+place.
+
+#### Upgrading an Ubuntu system
+
+Assuming the system was installed by following the [CC 2.x Ubuntu installation guide](https://github.com/01org/cc-oci-runtime/blob/master/documentation/Installing-Clear-Containers-on-Ubuntu.md):
+
+1. Perform the following cleanup steps:
+   ```bash
+   $ sudo rm -f /etc/systemd/system/docker.service.d/clr-containers.conf
+   $ sudo apt-get purge -y docker-engine
+   $ sudo rm -f /etc/apt/sources.list.d/cc-oci-runtime.list
+   ```
+
+1. Follow the [CC 3.x Ubuntu installation guide](docs/ubuntu-installation-guide.md).
+
+#### Upgrading a Fedora system
+
+Assuming the system was installed by following the [CC 2.x Fedora installation guide](https://github.com/01org/cc-oci-runtime/blob/master/documentation/Installing-Clear-Containers-on-Fedora.md):
+
+1. Perform the following cleanup steps:
+   ```bash
+   $ sudo rm -f /etc/systemd/system/docker.service.d/clr-containers.conf
+   $ sudo rm -f /etc/yum.repos.d/home:clearlinux:preview:clear-containers-2.1.repo
+   ```
+
+1. Follow the [CC 3.x Fedora installation guide](docs/fedora-installation-guide.md).
+
+## Unsupported operations and configurations
+
+### CentOS
+
+Since the CentOS installation instructions require the user to manually
+build and install various packages, upgrading is not supported on this
+platform.
+
+### Running 2.x and 3.x on the same system
+
+Running both 2.x and 3.x on the same system is explicitly not supported
+due to the incompatible elements between the two systems.
+
+If you are using the pre-built packages, installing the 3.x elements
+will remove the 2.x ones (see [Systems using pre-built
+packages](systems-using-pre-built-packages)).
+
+### Configuration files
+
+The 2.x runtime supported three different configuration files
+(`vm.json`, `hypervisor.args` and `cc-oci-runtime.sh.cfg` for the
+`cc-oci-runtime.sh` helper script).
+
+The 3.x runtime uses a single configuration file (`configuration.toml`).
+
+Note that none of the 2.x configuration files are migrated or read by
+the 3.x runtime. Any changes made to the 2.x configuration files will
+**not** be carried over to the 3.x runtime.
+
+### Downgrading
+
+Systems using packaged versions of 3.x cannot be downgraded to 2.x
+simply by following the 2.x installation instructions.
+
+However, it should be possible to switch back to 2.x by first fully
+removing 3.x and then following the 2.x installation instructions.
+
+Using 2.x is now discouraged given that it is no longer being maintained
```

Comment:
Nit: it is being maintained (critical fixes), but not developed (no new features) - maybe just semantics - let's check for the correct wording etc.

Reply:
Fixing...